types/testingutils/beacon_node.go

```diff
@@ -129,33 +131,33 @@ var TestingBeaconBlock = &bellatrix.BeaconBlock{
 			SyncCommitteeBits:      bitfield.NewBitvector512(),
 			SyncCommitteeSignature: phase0.BLSSignature{},
 		},
-		ExecutionPayload: &bellatrix.ExecutionPayload{
+		ExecutionPayload: &deneb.ExecutionPayload{
 			ParentHash:    phase0.Hash32{},
 			FeeRecipient:  bellatrix.ExecutionAddress{},
-			StateRoot:     phase0.Hash32{},
-			ReceiptsRoot:  phase0.Hash32{},
+			StateRoot:     phase0.Root{},
+			ReceiptsRoot:  phase0.Root{},
 			LogsBloom:     [256]byte{},
 			PrevRandao:    [32]byte{},
 			BlockNumber:   100,
 			GasLimit:      1000000,
 			GasUsed:       800000,
 			Timestamp:     123456789,
-			BaseFeePerGas: [32]byte{},
+			BaseFeePerGas: &uint256.Int{37},
 			BlockHash:     phase0.Hash32{},
 			Transactions:  Transactions,
 		},
 	},
 }
 
-var TestingBlindedBeaconBlock = func() *apiv1bellatrix.BlindedBeaconBlock {
+var TestingBlindedBeaconBlock = func() *apiv1deneb.BlindedBeaconBlock {
```

Comment:
This section of code appears to be unused now. It can be safely deleted as we have transitioned to using the `go-eth2-client` blocks, as specified in our `beacon_node_consts.go`. For reference, see [beacon_node_consts](https://github.com/bloxapp/ssv-spec/blob/3d553a7de7b6ddeaf46bcc3a6d86258e20ae2730/types/testingutils/beacon_node_consts.go).